pull.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import plotly.offline as py
+import plotly.graph_objs as go
+
+index = ['MU','GE','AMD','BAC']
+
+
+def main():
+    stocks = pd.DataFrame()
+    i = 0
+    while i < len(index):
+        r = requests.get('https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=' + str(
+            index[i]) + '&datatype=csv&apikey=603MDIJGG9TGNV60')
+        file = open('C:\\Users\\brain\\PycharmProjects\\Stocks\\pull.csv', "w+")
+        file.write(r.text)
+        r.content.decode()
```

Comment:
Look into creating a new dataframe directly from using r.content.decode() instead of creating a CSV file.